pkg/armrpc/hostoptions/providerconfig.go

```diff
@@ -21,19 +21,19 @@ import (
 	profilerprovider "github.com/radius-project/radius/pkg/profiler/provider"
 	"github.com/radius-project/radius/pkg/trace"
 	"github.com/radius-project/radius/pkg/ucp/config"
-	"github.com/radius-project/radius/pkg/ucp/dataprovider"
-	qprovider "github.com/radius-project/radius/pkg/ucp/queue/provider"
-	sprovider "github.com/radius-project/radius/pkg/ucp/secret/provider"
+	"github.com/radius-project/radius/pkg/ucp/databaseprovider"
+	"github.com/radius-project/radius/pkg/ucp/queue/queueprovider"
+	"github.com/radius-project/radius/pkg/ucp/secret/secretprovider"
```

Comment:
So, we renamed the **queue/provider** and **secret/provider** to **queue/queueprovider** and **secret/secretprovider** but kept **databaseprovider** under **ucp**. 🤨

Reply:
I'm planning to rearrange these in the next PR 👍